wonderous-book/lib.typ

```diff
@@ -86,7 +80,19 @@
       // Find the heading of the section we are currently in.
       let before = query(selector(heading).before(here()))
       if before != () {
-        align(right, text(0.95em, smallcaps(before.last().body)))
+        set text(0.95em)
+        let header = smallcaps(before.last().body)
+        let title = smallcaps(title)
+        let author = text(style: "italic", author)
+        grid(
+          columns: (auto, 1fr, auto),
```

Comment:
I think if we did something like `1fr, 10fr, 1fr` here, there'd be no need for the hide and things would stay centered. Or we could just use multiple `place`s to prevent the centering from breaking due to the numbers at the side.

Reply:
Sorry for the slow response, I've been flat out trying to finish a thesis (in typst no doubt :smile:).

Doing `1fr, 10fr, 1fr` also has the benefit that it puts a little space on either side of the numbers which I think is desirable in the edge case of a really long title or author.